api/v1alpha1/virtualmachineimage_types.go

```diff
@@ -119,6 +134,7 @@ func (vmImage *VirtualMachineImage) SetConditions(conditions Conditions) {
 // +kubebuilder:resource:scope=Cluster,shortName=vmimage
 // +kubebuilder:storageversion
 // +kubebuilder:subresource:status
+// +kubebuilder:printcolumn:name="ImageSourceId",type="string",JSONPath=".status.imageInfo.imageSourceId"
```

Comment:
The imageSourceId is a uuid, which isn't really human friendly (and kubebuilder:printcolumn:name are meant for human consumption). Perhaps the ImageName would be better here, thoughts?

Reply:
IMO, for now developers can only know the UUID of which content source they have access to. Maybe print out the ImageSourceId can help developers figure out what vm images belong to the attached content source? 
We only append several random bits to the vm image name, so that it should not lose much readability.  Will it be a little bit verbose if we add another column to print the ImageName?

Reply:
Ah okay, so this column is meant for the vmimage's content source. Even in that case, knowing a uuid still isn't all that useful. How about introducing status.imageInfo.imageSourceName (friendly content library name?), and then displaying that instead of the uuid. (while still keeping status.imageinfo.imageSourceId, but not displaying it as a column)

Reply:
Yes a uuid is not human friendly. :(  But we'll have some issues if we don't print the uuid but print the status.imageInfo.imageSourceName instead.
1.  A developer can't match the content source k8s resource name (uuid) and its display name. (This can be done by adding  content source display name field to ContentSourceBinding and ContentSource CRD. )
2. We can't ensure that a VI admin won't create two content libraries with the same name. This scenario is allowed in the vCenter, while UUID is guaranteed to be unique.